marketplace/commercetools-products/src/dialog.js

```diff
@@ -0,0 +1,91 @@
+/* global CTPicker */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Button } from '@contentful/forma-36-react-components';
+
+export class CommerceToolsDialog extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.pickerContainerRef = React.createRef();
+    this.picker = null;
+
+    this.onCancel = this.onCancel.bind(this);
```

Comment:
I think you should be able to define these methods as properties ( `onCancel = () => { }` in the class definition) so you don't need to bind.